Makefile

```diff
@@ -12,16 +12,19 @@ endif
 
 notebookImageName=$(repository)/quantum-serverless-notebook
 rayNodeImageName=$(repository)/quantum-serverless-ray-node
-gatewayImageName=$(repository)/quantum-serverless-gateway
+#gatewayImageName=$(repository)/quantum-serverless-gateway
+gatewayImageName=qiskit/quantum-serverless-gateway
+#proxyImageName=$(repository)/quantum-serverless-proxy
+proxyImageName=qiskit/quantum-serverless-proxy
```

Comment:
assuming these were local testing too... though also wondering if anyone uses the makefile to build images?